src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -5,418 +5,41 @@
   useGetAllFeatures
 } from 'services/cf'
-import { useStrings, String } from 'framework/strings'
-import { ResourceType } from '@rbac/interfaces/ResourceType'
-import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
-import RBACTooltip from '@rbac/components/RBACTooltip/RBACTooltip'
-import { usePermission } from '@rbac/hooks/usePermission'
-import { UseToggleFeatureFlag, useToggleFeatureFlag } from '@cf/hooks/useToggleFeatureFlag'
-import { VariationTypeIcon } from '@cf/components/VariationTypeIcon/VariationTypeIcon'
-import { VariationWithIcon } from '@cf/components/VariationWithIcon/VariationWithIcon'
+import { useStrings } from 'framework/strings'
+import { useToggleFeatureFlag } from '@cf/hooks/useToggleFeatureFlag'
 import ListingPageTemplate from '@cf/components/ListingPageTemplate/ListingPageTemplate'
 import { useEnvironmentSelectV2 } from '@cf/hooks/useEnvironmentSelectV2'
 import { CFEnvironmentSelect } from '@cf/components/CFEnvironmentSelect/CFEnvironmentSelect'
 import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
-import {
-  CF_DEFAULT_PAGE_SIZE,
-  featureFlagHasCustomRules,
-  getDefaultVariation,
-  getErrorMessage,
-  isFeatureFlagOn,
-  useFeatureFlagTypeToStringMapping
-} from '@cf/utils/CFUtils'
-import { FlagTypeVariations } from '@cf/components/CreateFlagDialog/FlagDialogUtils'
+import { CF_DEFAULT_PAGE_SIZE } from '@cf/utils/CFUtils'
 import FlagDialog from '@cf/components/CreateFlagDialog/FlagDialog'
-
-import SaveFlagToGitModal from '@cf/components/SaveFlagToGitModal/SaveFlagToGitModal'
-import { GIT_COMMIT_MESSAGES } from '@cf/constants/GitSyncConstants'
 import GitSyncActions from '@cf/components/GitSyncActions/GitSyncActions'
-import { GitDetails, GitSyncFormValues, GIT_SYNC_ERROR_CODE, UseGitSync } from '@cf/hooks/useGitSync'
-import { useGovernance, UseGovernancePayload } from '@cf/hooks/useGovernance'
-import usePlanEnforcement from '@cf/hooks/usePlanEnforcement'
-import FlagOptionsMenuButton from '@cf/components/FlagOptionsMenuButton/FlagOptionsMenuButton'
-import { FeatureIdentifier } from 'framework/featureStore/FeatureIdentifier'
-import { useFeature } from '@common/hooks/useFeatures'
+import { useGovernance } from '@cf/hooks/useGovernance'
 import { useFeatureFlag } from '@common/hooks/useFeatureFlag'
-import { FeatureWarningTooltip } from '@common/components/FeatureWarning/FeatureWarningWithTooltip'
 import { FeatureFlag } from '@common/featureFlags'
 import { useFFGitSyncContext } from '@cf/contexts/ff-git-sync-context/FFGitSyncContext'
 import type { FilterProps } from '@cf/components/TableFilters/TableFilters'
-import { FeatureFlagStatus, FlagStatus } from './FlagStatus'
-import { FlagResult } from './FlagResult'
+import routes from '@common/RouteDefinitions'
+import AllEnvironmentsFlagsListing from '@cf/pages/feature-flags/components/AllEnvironmentsFlagsListing'
+import FeatureFlagsListing from '@cf/pages/feature-flags/components/FeatureFlagsListing'
```

Comment:
Not a blocker, but it might make it more obvious that this is using the pages own components if we make these relative. Also keeps in the same style as the other page-specific components below.
```suggestion
import AllEnvironmentsFlagsListing from './components/AllEnvironmentsFlagsListing'
import FeatureFlagsListing from './components/FeatureFlagsListing'
```